include/modem/location.h

```diff
@@ -465,6 +465,11 @@ struct location_config {
 	 * Default value is 300000 (5 minutes). It is applied when
 	 * @ref location_config_defaults_set function is called and can be changed
 	 * at build time with CONFIG_LOCATION_REQUEST_DEFAULT_TIMEOUT configuration.
+	 *
+	 * This is intended to be a safety timer preventing location request from getting stuck
+	 * in any of its phases because parts of the method specific functionality are not
+	 * covered by method specific timeouts. A rather large value should be used which is
+	 * even two times or more of the sum of method specific timeouts.
```

Comment:
```suggestion
	 * This is intended to be a safety timer preventing location request from getting stuck
	 * in any of its phases, because parts of the method-specific functionality are not
	 * covered by method-specific timeouts. You should use a rather large value that
	 * can be two or more times of the sum of method-specific timeouts.
```

Reply:
I've further modified this to use "one minute or more" rather than "two or more times". Please check how this is best said. This is technically more accurate.